Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -927,6 +927,18 @@ double numberCounts(const Histogram &histogram, const double xmin, const double
     total += std::fabs(histogram.y()[i]);
   return total;
 }
+
+//----------------------------------------------------------------------------------------------
+/** Get number of vector elements falling into specified range
+ * @param vector :: vector of type T
+ * @param range :: range boundaries
+ * @return :: number of elements
+ */
+template <typename T> size_t numberElements(const std::vector<T> &elements, const std::pair<T, T> &range) {
+  const auto leftIter = std::lower_bound(elements.cbegin(), elements.cend(), range.first);
+  const auto rightIter = std::upper_bound(elements.cbegin(), elements.cend(), range.second);
+  return size_t(rightIter - leftIter);
```

Comment:
This might be clearer if it used [std::distance](https://en.cppreference.com/w/cpp/iterator/distance)
```suggestion
  return std::distance(leftIter, rightIter);
```

Reply:
Updated code.